.evergreen/integration-tests.sh

```diff
@@ -155,15 +146,26 @@ case "$OS" in
       ;;
 esac
 
-sleep 15
-echo "Checking that mongo-orchestration is running"
-curl http://localhost:8889/ -sS --max-time 120 --fail | python -m json.tool
-
+echo "Waiting for mongo-orchestration to start..."
+wait_for_mongo_orchestration() {
+   for i in $(seq 300); do
+      # Exit code 7: "Failed to connect to host".
+      if curl -s "localhost:$1"; test $? -ne 7; then
+         return 0
+      else
```

Comment:
```suggestion
      else
        # Print newline to separate output of curl.
        echo ""
```

The response [appears not to have a newline](https://evergreen.mongodb.com/lobster/evergreen/task/mongo_c_driver_asan_ubuntu_test_asan_4.0_replica_set_noauth_nosasl_nossl_patch_d1134294420514f4308507ff5b2a5f4764d93c73_639ce8202a60ed35c2dec6bb_22_12_16_21_50_25/0/task#bookmarks=0%2C10357&l=1&shareLine=132).

```
[2022/12/16 22:01:54.065] {"service": "mongo-orchestration", [...] "rel": "get-sharded-clusters"}]}Waiting for mongo-orchestration to start... done.